### PR TITLE
Ac 7

### PR DESCRIPTION
**Pull request by jatin471**
Added new examination folder in Side Drawer. 
Added Examination Folder…
… and its files :- 
         AnnounceGrade.dart ,
         ExaminationHomePage.dart ,
          GenerateResult.dart , 
         VerifyResult.dart. 

 Added A Custom App Bar file/function for Different Change Title with Custom name .

In api.dart added 3 more apis link so to load our data from backend

### CODE REVIEW

**Comment by jatin471** [comment]
@prajjwalkapoor

**Comment by prajjwalkapoor** [comment]
@Kushagra-Yadav 
@abhi87374⁰
@Kunal-2417

Do check if we can integrate this with the main.

**Comment by jatin471** [comment]
Added Verification Checkboxes and also wrote APIs for it and also resolved merge errors

File: ios/RunnerTests/RunnerTests.swift [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Flutter
+import UIKit
+import XCTest
+
+class RunnerTests: XCTestCase {
```
Comment (AgPriyanshu18) [inline ios/RunnerTests/RunnerTests.swift:5]: This Folder should not be present

**Review by AgPriyanshu18 — CHANGES_REQUESTED** [review]

**Comment by AgPriyanshu18** [comment]
@jatin471 Please update distribution URL in gradle-wrapper-properties

File: ios/RunnerTests/RunnerTests.swift (hunk 1 above)
Comment (jatin471) [reply ios/RunnerTests/RunnerTests.swift]: Removed

**Review by AgPriyanshu18 — APPROVED** [review]

**Pull request merged**